tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -162,6 +166,13 @@ def test_version_returns_results(client):
     assert not version_info["serving"] == "not configured"
 
 
+def test_list_feature_sets_when_auth_enabled_should_raise(enable_auth):
```

Comment:
I wanted to use `pytest.mark`. It was making it mutually exclusive. Instead of running this along with others when auth is enabled. It was either running this or others based on `-m` options. I didn't find options to pass multiple markers.